14a_alpha_diversity.Rmd

```diff
@@ -109,6 +109,29 @@ se <- mia::estimateDiversity(se,
 head(colData(se)$shannon)
 ```
 
+Alpha diversities can be visualized with boxplot. Here, Shannon index is compared 
+between different sample type groups. Individual data points are visualized by 
+plotting them as points. With more data points, test `geom_jitter` function
+instead of `geom_point`.
+
+```{r visualize-shannon}
+library(ggplot2)
+ggplot(as.data.frame(colData(se)), aes(x = SampleType, y = shannon)) +
+  # Outliers are removed, because otherwise each data point would be plotted twice; 
+  # as an outlier of boxplot and as a point of dotplot.
+  geom_boxplot(outlier.shape = NA) + 
+  geom_point() + 
```

Comment:
wouldn't `geom_jitter` work better? The jitter width could be set to 0.5 or something suitable.

Reply:
Changed to jitter